tests/storage-vm

```diff
@@ -599,6 +599,53 @@ for poolDriver in $poolDriverList; do
                 [ "$(($(lxc exec v1 -- blockdev --getsize64 /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_root) / GiB))" -eq "16" ]
         fi
 
+        lxc stop -f v1
+        # Set 'volume.size' and create new VM
+        lxc storage set "${poolName}" volume.size="7GiB"
+        lxc init -s "${poolName}" testimage v3
+
+        # Get instance root disk sizes
+        [ "$(lxc info v1 | awk '/Disk total:/ {found=1} found && /root:/ {print $2; exit}')" = "10GiB" ]
+        [ "$(lxc info v3 | awk '/Disk total:/ {found=1} found && /root:/ {print $2; exit}')" = "7GiB" ]
+
+        # Check volume 'size' and volume state also show correct total size.
+        [ "$(lxc storage volume get virtual-machine/v1 size)" = "10GiB" ]
+        [ "$(lxc storage volume info virtual-machine/v1 | grep "Total:")"  = "10GiB" ]
+        [ "$(lxc storage volume get virtual-machine/v3 size)" = "7GiB" ]
+        [ "$(lxc storage volume info virtual-machine/v3 | grep "Total:")"  = "7GiB" ]
```

Comment:
Quick drive-by observation. All those `lxc storage volume` are seemingly missing the pool name.